packages/server/src/app/db/works/works.service.ts

```diff
@@ -388,46 +388,36 @@ export class WorksService {
      * Fetches all new published works by newest first.
      */
     async fetchNewPublishedWorks(contentFilter: models.ContentFilter, pageNum: number): Promise<PaginateResult<documents.WorkDocument>> {
-        if (contentFilter === models.ContentFilter.Everything) {
-            return await this.workModel.paginate({
-                'audit.published': models.ApprovalStatus.Approved,
-                'audit.isDeleted': false,
-            }, {
-                sort: {'audit.publishedOn': -1},
-                page: pageNum,
-                limit: 15
-            });
-        } else if (contentFilter === models.ContentFilter.MatureEnabled) {
-            return await this.workModel.paginate({
-                'audit.published': models.ApprovalStatus.Approved,
-                'audit.isDeleted': false,
-                $or: [{'meta.rating': models.ContentRating.Everyone}, {'meta.rating': models.ContentRating.Teen}, {'meta.rating': models.ContentRating.Mature}]
-            }, {
-                sort: {'audit.publishedOn': -1},
-                page: pageNum,
-                limit: 15
-            });
-        } else if (contentFilter === models.ContentFilter.ExplicitEnabled) {
-            return await this.workModel.paginate({
-                'audit.published': models.ApprovalStatus.Approved,
-                'audit.isDeleted': false,
-                $or: [{'meta.rating': models.ContentRating.Everyone}, {'meta.rating': models.ContentRating.Teen}, {'meta.rating': models.ContentRating.Explicit}]
-            }, {
-                sort: {'audit.publishedOn': -1},
-                page: pageNum,
-                limit: 15
-            });
-        } else {
-            return await this.workModel.paginate({
-                'audit.published': models.ApprovalStatus.Approved,
-                'audit.isDeleted': false,
-                $or: [{'meta.rating': models.ContentRating.Everyone}, {'meta.rating': models.ContentRating.Teen}]
-            }, {
-                sort: {'audit.publishedOn': -1},
-                page: pageNum,
-                limit: 15
-            });
+        let query = {'audit.published': models.ApprovalStatus.Approved, 'audit.isDeleted': false};
+        let paginateOptions = {sort: {'audit.publishedOn': -1}, page: pageNum, limit: 15};
+
+        switch (contentFilter) {
+            case models.ContentFilter.Everything: 
+                query = query;
+                break;
```

Comment:
if you wanted, you could even remove this arm entirely

Reply:
i'll merge this and change that up in the next PR